src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
+
+        return Colors.values()[index].toString();
```

Comment:
```suggestion
    private Random random = new Random();
    
    public String getRandomColor() {
        int index = random.nextInt(Colors.values().length);
        return Colors.values()[index].name();
```

Reply:
as mentioned in checklist, let's avoid static methods, and calling .name() on enum is preferable in comparison with .toString(). Random - same as in Lottery class
![image](https://github.com/mate-academy/jv-lottery/assets/49767049/67d21643-a027-4d40-a733-e849f37652a5)
![image](https://github.com/mate-academy/jv-lottery/assets/49767049/6c3bb73e-82a7-4bf8-ab23-bfba96554e35)